integration-libs/cdc/root/guards/cdc-user.guard.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { CanActivate, Router, UrlTree } from '@angular/router';
+import {
+  GlobalMessageService,
+  GlobalMessageType,
+  SemanticPathService,
+} from '@spartacus/core';
+import { UserGuard } from '@spartacus/organization/administration/core';
+import { CdcB2BUserService } from '../../organization/administration/public_api';
```

Comment:
it will give Entry point @spartacus/cdc/organization/administration has a circular dependency on @spartacus/cdc/root error during build, hence not importing like @spartacus/cdc/organization/administration